.github/workflows/test-query-engine.yml

```diff
@@ -104,7 +104,7 @@ jobs:
       single_threaded: true
 
   cockroachdb-push:
-    if: github.event_name == 'push'
+    #if: github.event_name == 'push'
```

Comment:
Temporarily enabled QE tests on CocroachDB for PR testing. This condition must be re-enabled before merging this PR.